jobs/quoscient_cybersecurity_germany_permanent.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Senior Python Developer
+company: QuoScient GmbH
+url: http://quoscient.io
+location: Frankfurt am Main, Germany
```

Comment:
You can give a more precise address (even including a postal code) - this will allow candidates to exactly locate you neighborhood.